DiamondLightSource/python-artemis
# 287: Tidy up the eiger

Fixes #287 

To test:
* Confirm on the original issue that all queries have been resolved
* Confirm tests still pass
